src/database/models/users.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+import { Model } from 'sequelize';
+export default (sequelize, DataTypes) => {
+  class users extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  users.init({
+    fullname: DataTypes.STRING,
+    username: DataTypes.STRING,
+    email: DataTypes.STRING,
+    role: DataTypes.STRING,
```

Comment:
there must be specific values for `role field`

Reply:
hi, thank you for the feedback but in the task they precise to say I have to put dummy data in the database that's why I put that one.